test/models/CurrentAndPreviousYearsEligibilityTest.scala

```diff
@@ -23,31 +23,27 @@ import org.mockito.Mockito._
 
 class CurrentAndPreviousYearsEligibilityTest extends UnitSpec with MockitoSugar {
 
-  val currentYear = 2019
-  val previousTaxYear = TaxYear(2018)
-  val mockData = mock[RegistrationFormInput]
-  val mockTimeService = mock[TimeService]
-  when(mockTimeService.getCurrentTaxYear).thenReturn(currentYear)
-
+  val currentYear: Int = 2019
```

Comment:
could we use TaxYear here to get the current year or is this something to do with the uprating?

Reply:
good point, changed